### ISSUE TITLE
Why encoded?

### ISSUE THREAD

**Issue by shmert** [jshor/angular-addtocalendar#7]
Downloaded the .ics from the demo file, and I cannot add it to the calendar. The file contents are URL encoded.

BEGIN%3AVCALENDAR%0AVERSION%3A2%0ABEGIN%3AVEVENT%0ACLASS%3APUBLIC%0ADESCRIPTION%3ACelebrate%20the%20independence%20of%20the%20United%20States%20with%20fireworks%20in%20one%20of%20the%20greatest%20cities%20in%20the%20world.%0ADTSTART%3BVALUE%3DDATE%3A20150704T190000%0ADTEND%3BVALUE%3DDATE%3A20150704T210000%0ALOCATION%3ABattery%20Park%20City%2C%20New%20York%2C%20NY%0ASUMMARY%3BLANGUAGE%3Den-us%3AFourth%20of%20July%20Fireworks%0ATRANSP%3ATRANSPARENT%0AEND%3AVEVENT%0AEND%3AVCALENDAR

**Comment by jshor** [comment]
The original method of downloading the .ics file was to navigate to a url with the `data:` prefix, but it was discovered that not all browsers support that.

The method was then switched to downloadjs but the function doesn't require it to be url-encoded.

**tl;dr**: it's fixed it in v1.1.3. My bad.
